docs/server-backup-version-comparison.md

```diff
@@ -1,12 +1,10 @@
-<!--- will this be in 8.1?
- --->
-
 # Server version and backup version comparison
 
 A MySQL change to a feature, such as the structure of a redo log
 record, can cause older versions of Percona XtraBackup to fail. To
 ensure that you can back up and restore your data, use a Percona
-XtraBackup version that is equal to or above your source server version.
+XtraBackup version that is equal to or within your source server major version. For example, if you use Percona
```

Comment:
Maybe add a reminder that 8.1 only works with 8.1?

Reply:
Updated